tests/tool-brew.yaml

```diff
@@ -7,7 +7,13 @@
   command: [brew, doctor]
   assert:
   - stderr.indexOf("error") == -1
+  - status == 0
 - desc: it should have valid configuration
   command: [brew, config]
   assert:
   - status == 0
+- desc: it should have cmake
+  command: [cmake, --version]
+  assert:
+  - stderr.indexOf("cmake version 3.25.") == -1
```

Comment:
3.25 is not neccessary since brew always install latest

Reply:
Fixed in https://github.com/gitpod-io/workspace-images/compare/c83a0a56faf9c18d36b644b599d2863f128f0ab2..1c9acdf5883b3675d42a73d38535465673befd11